conduit/plugins/importers/filereader/sample.yaml

```diff
@@ -1,14 +1,14 @@
-name: file_reader
-config:
-  # BlocksDir is the path to a directory where block data should be stored.
-  # The directory is created if it doesn't exist.
-  block-dir: "/path/to/directory"
-  # RetryDuration controls the delay between checks when the importer has
-  # caught up and is waiting for new blocks to appear.
-  retry-duration: "5s"
-  # RetryCount controls the number of times to check for a missing block
-  # before generating an error. The retry count and retry duration should
-  # be configured according the expected round time.
-  retry-count: 5
-  # FilenamePattern is the format used to find block files. It uses go string formatting and should accept one number for the round.
-  filename-pattern: "%[1]d_block.json"
+  name: file_reader
+  config:
+    # BlocksDir is the path to a directory where block data should be stored.
+    # The directory is created if it doesn't exist.
+    block-dir: "/path/to/directory"
+    # RetryDuration controls the delay between checks when the importer has
+    # caught up and is waiting for new blocks to appear.
+    retry-duration: "5s"
+    # RetryCount controls the number of times to check for a missing block
+    # before generating an error. The retry count and retry duration should
+    # be configured according the expected round time.
+    retry-count: 5
+    # FilenamePattern is the format used to find block files. It uses go string formatting and should accept one number for the round.
+    filename-pattern: "%[1]d_block.json"
```

Comment:
nit: do we need space before these lines?

Reply:
For it to be formatted correctly we do.